### ISSUE TITLE
Reproducing issues on cross-domain setting.

### ISSUE THREAD

**Issue by lwmming** [HashmatShadab/APR#1]
Thank you for your insightful work! 
As your suggestion, in cross-domain setting, I train a single surrogate model by Rotation mode on CoCo-2017(41k). But I get following Top-1 (%) acc:

Resnet-152,37.76
VGG-19,27.48
Inception-V3,38.82
DenseNet-121,35.22
WRN-101,32.62
MobileNet-v2,21.72
senet,47.62
Average accuracy,34.46

which is far worse than 23.31 reported in paper. Could you help me?
- I use the [COCO2017 Test images (41K/6GB)](http://images.cocodataset.org/zips/test2017.zip) to train the  surrogate model. I am not sure if it's the same as your training data.
- I train the surrogate model for 50 epoch (i.e., rotate_0.pth~rotate_49.pth) and use rotate_49.pth to generate adversarial examples.
- The batchsize is set as 64 as default in your codes.

Thank you very much! Looking forward to your reply~
